includes/admin/class-rop-admin.php

```diff
@@ -797,6 +800,20 @@ public function rop_dismiss_linkedin_api_v2_notice() {
 
 	}
 
+
+	/**
+	 * Disable Cron Jobs on refresh if remove_cron() method was called
```

Comment:
Let's add the phpdoc comment here for `since` 8.5.0 and if you think it's needed here `see`. There are a few more instances where the `since` wasn't added so lets add it in those areas as well

Reply:
Added the tag since to the methods created by me in the branch.